### DISCUSSION TITLE
turn on RX2 and TX2 on Matek F405 wte

### DISCUSSION THREAD

**Discussion by rich6777** [iNavFlight/inav#8593] [General]
Hi every one.... Try connect Rasberry Pi4 to Matek F405 wte, but catn do it...  I connect Pi4 to matek on RX2, TX2....But there is no reaction... How i can turn on RX2, TX2 on Matek F405 wte ? 
Should i turn on some ports in iNav ??? Spend a lot time for it, but still no reaction from RX2 TX2...

**Comment by stronnag** [comment]
You need to set a function for the port (e.g. MSP).

**Comment by rich6777** [comment]
what  function  ?

MSP what it is ?

вт, 29 нояб. 2022 г. в 16:11, Jonathan Hudson ***@***.***>:

**Reply by MrD-RC** [reply]
It would be whatever your Pi is expecting.
